src/CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ if(NOT (CMAKE_BUILD_TYPE IN_LIST SUPPORTED_BUILD_TYPES))
 			"Unknown CMAKE_BUILD_TYPE: ${CMAKE_BUILD_TYPE}. Options are: ${SUPPORTED_BUILD_TYPES}.")
 endif()
 
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} $ENV{CFLAGS} $ENV{CPPFLAGS}")
```

Comment:
but CPPFLAGS is for C++ only?

Reply:
I think this was initial mistake which got into final patch, I removed CPPFLAGS.